smoke_tests.sh

```diff
@@ -2,3 +2,4 @@
 set -ex
 
 [ $(curl --write-out %{http_code} --silent --output /dev/null https://$(hostname)/) == 200 ]
+[ $(curl --write-out %{http_code} --silent --output /dev/null https://$(hostname)/api/v2/articles) == 200 ]
```

Comment:
if you can think of other lightweight URLs that should always be working in any environment and any situation, please add them here